src/coreComponents/mesh/EmbeddedSurfaceSubRegion.hpp

```diff
@@ -40,10 +40,10 @@ class EmbeddedSurfaceSubRegion : public ElementSubRegionBase
 public:
 
   /// Embedded surface element to nodes map type
-  using NodeMapType = InterObjectRelation< array2d< localIndex, cells::NODE_MAP_PERMUTATION > >;
+  using NodeMapType = InterObjectRelation< ArrayOfArrays< localIndex > >;
 
   /// Embedded surface element to edges map type
-  using EdgeMapType = InterObjectRelation< ArrayOfArrays< localIndex > >;
+  using EdgeMapType = InterObjectRelation< array1d< array1d< localIndex > > >;
```

Comment:
Tbh, this map is not used by this subRegion. They type alias is defined because otherwise I have issues in the `[DofManagerHelper.hpp ](https://github.com/GEOSX/GEOSX/blob/develop/src/coreComponents/linearAlgebra/DofManagerHelpers.hpp)` at line 66.

Reply:
We are trying to remove all `array1d<array1d<>>` from the code. Can you go back to this original code?